### DISCUSSION TITLE
インシデント一覧ページで原因の表示

### DISCUSSION THREAD

**Discussion by gr1m0h** [topotal/waroom-roadmap#68] [Feature Request]
インシデント一覧ページ(https://app.waroom.com/organizations/<organization_name>/incidents)では、インシデント名、ステータス、SEV、サービス名、起票者が表示されているが、原因・ラベルについて表示されていないので表示していただきたい

**Comment by sawa-zen** [comment]
任意ラベルに関しては一覧ページにすでに表示されると思います。もし表示されなければ教えていただけると助かります。原因に関しては現在表示できていないので検討します！

**Reply by gr1m0h** [reply]
ありがとうございます！
確認漏れだと思うので、原因を対象としたものにタイトルも変えます！

**Reply by sawa-zen** [reply]
@gr1m0h 
原因ラベルをあらたに一覧に表示するように修正しました！
![CleanShot 2024-03-26 at 14 38 25@2x](https://github.com/topotal/waroom-roadmap/assets/3971271/e152062c-79b1-4b04-8fb5-7755b4e03b1c)

**Reply by gr1m0h** [reply]
@sawa-zen 
確認しました！ありがとうございます！
問題ないと思うのでこのDiscustionをCloseしても良いかと思いました

**Reply by sawa-zen** [reply]
確認ありがとうございます！ではこちらクローズしますね 😃
